molgenis/molgenis-emx2
# poc(beacon): configurable templates for beacon api

#4533 

### What are the main changes you did
- Added a templates table in the system schema
- Added a templates tab in "Admin > Templates" to configure the templates 
- Load default templates in db from templates at classpath.
- Beacon api looks for a custom template or else picks the default template.

### How to test
- Go to https://preview-emx2-pr-4536.dev.molgenis.org/apps/central/#/
- Log in as admin
- Create a schema with the BEACON template or other beacon compatible schema
- Go to Admin > Templates
- Change a template and link it to the created schema e.g. the individuals template
- go to: https://preview-emx2-pr-4536.dev.molgenis.org/<schema-name>/api/beacon/individuals
- The response should now include the changes from the template 

### Checklist
